src/charts/bar.tsx

```diff
@@ -13,15 +13,15 @@ interface BarOptions {
   colors: string[];
   setColor: (color: string[]) => void;
   barGrouping: Dx.BarGroupingType;
-  dimensions: object[];
-  metrics: object[];
+  dimensions: Dx.Dimension[];
+  metrics: Dx.Metric[];
 }
 
 export const semioticBarChart = (
   data: Dx.Datapoint[],
   schema: Dx.Schema,
   options: BarOptions,
-  colorHashOverride?: { key?: string },
+  colorHashOverride?: any,
```

Comment:
Maybe this was supposed to be something like `{ [key?: string]: string }`

Reply:
Yeeeup